tests/python_client/testcases/test_query.py

```diff
@@ -5598,3 +5598,22 @@ def test_query_text_match_with_unsupported_tokenizer(self):
             check_task=CheckTasks.err_res,
             check_items=error,
         )
+
+
+class TestQueryFunction(TestcaseBase):
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_function_calls(self):
+        """
+        target: test query data
+        method: create collection and insert data
+                query with mix call expr in string field and int field
+        expected: query successfully
+        """
+        collection_w, vectors = self.init_collection_general(prefix, insert_data=True,
+                                                             primary_field=ct.default_string_field_name)[0:2]
+        res = vectors[0].iloc[:, 1:3].to_dict('records')
+        output_fields = [default_float_field_name, default_string_field_name]
+        for mixed_call_expr in ["not empty(varchar) && int64 >= 0", "starts_with(varchar, varchar) && int64 >= 0"]:
+            collection_w.query(mixed_call_expr, output_fields=output_fields,
+                               check_task=CheckTasks.check_query_results, check_items={exp_res: res})
+
```

Comment:
add a negative case for not supported function

Reply:
Done. I also added `EmPty` and `EMPTY` for checking case-insensitive function calls